src/cbmc/symex_bmc.cpp

```diff
@@ -110,27 +110,44 @@ bool symex_bmct::get_unwind(
 {
   const irep_idt id=goto_programt::loop_id(*source.pc);
 
-  // We use the most specific limit we have,
-  // and 'infinity' when we have none.
-
+  tvt unwind_decision;
```

Comment:
This is actually a bit of a misnomer, because `unwind_decision.is_true()` is interpreted as _don't_ unwind. The `abort` variable below makes this very obvious.

Reply:
`should_abort_unwind`?